### PR TITLE
started safety check

### PR DESCRIPTION
**Pull request by k2shah**
safety checklist for documentation 

quick start guide should be broken up a little maybe?

### CODE REVIEW

**Comment by exoticDFT** [comment]
What do you propose @k2shah ?

Currently, the plan was to have the safety checklist as an individual subsection of the quick start guide. In the sense that the main page (what you started to edit) of the quick start guide would have a short description of the checks you should perform and then an in depth page (needs to be created) of those checks. This way we have some guidance for those who already know what they are doing, but might not recall without a bit of reminder, and then a detailed page familiarizing those who are just starting, or those that need a more detailed reminder.

Did you want to break it up in a different way? I sketched this out in the OneNote notebook, so we can visualize the plan a bit more.

**Comment by k2shah** [comment]
The quick start guide mentions 3 major setup tasks. In it's current state it's not a quick start. I think we should mention the setup tasks at the top and then have an enumerated list of the steps you have to do to get the quad off the ground, with the links to each section for more detail 

Setup
- Ground Control/Mocap 
- Companion Computer
- Flight Control Unit

^ each of the following should have a brief description and then  links to a more detailed section (we have 2 of 3 so far)

Flight
1. Turn on mocap
2. Set ridge body of the quad with the correct orientation 
3. Turn on handheld control (link to that section)
4. Check lipo battery over, plug into quad 
5. Arm check with the controller (disarm after)
6. ssh into quad
7. run `quad_vrpn.launch`
8. run the controller (pose, vel, traj, custom, etc) (link to section)
9. review safety checklist (and have the checklist under this)

The final quick start guide would look like

setup
flight
safety checklist

could also interweave the safety checklist and the flight (rather pre-flight) together as they have a lot of overlap. I also think that quick start can be it's own high level header (in line with software and hardware).. I think we talked about this before (not sure what side i was on). with quick start as it's own header we can have the 3 setup subsections and maybe a this would be a good home for the trouble shooting section? 

^ could be better organized, but something like this.

File: docs/source/software/quick_start_guide.rst
Comment (exoticDFT) [inline docs/source/software/quick_start_guide.rst]: I would make this part a warning or important directive, so that it is highlighted. You should be able to do this with the following:

```
.. important::
    Make sure you ...
```

File: docs/source/software/quick_start_guide.rst
Comment (exoticDFT) [inline docs/source/software/quick_start_guide.rst]: Do we need to be specific about what "secured" means? Since for the rotors, it seems that twisting the bolts by hand isn't always enough for longer term use.

File: docs/source/software/quick_start_guide.rst
Comment (exoticDFT) [inline docs/source/software/quick_start_guide.rst]: I think the above Software 1. and 2. look good, but I'm not sure I specifically follow what you mean by 3. I think some elaboration on each of these points might be useful.

I think this checklist is great for running through to verify safety before flying. However, for all the points in this checklist, would it be beneficial to have an elaborated page of the safety guide? There, maybe pictures or elaboration could be useful for someone who has never used the system before. This could include the comment I made earlier about the meaning of "secured".

**Review by exoticDFT — CHANGES_REQUESTED** [review]
I think this checklist is looking good. I left a couple of comments below to help with some clarification. I figure we'll address the remaining points from the above discussion in the MSLQ-79.

File: docs/source/software/quick_start_guide.rst
Comment (k2shah) [reply docs/source/software/quick_start_guide.rst]: done

File: docs/source/software/quick_start_guide.rst
Comment (k2shah) [reply docs/source/software/quick_start_guide.rst]: done

File: docs/source/software/quick_start_guide.rst
Comment (k2shah) [reply docs/source/software/quick_start_guide.rst]: done

Comment (exoticDFT) [reply docs/source/software/quick_start_guide.rst]: This looks great with the directive!

File: docs/source/software/quick_start_guide.rst
Comment (exoticDFT) [reply docs/source/software/quick_start_guide.rst]: This is much more clear now!

File: docs/source/software/quick_start_guide.rst
Comment (exoticDFT) [inline docs/source/software/quick_start_guide.rst]: > I think the above Software 1. and 2. look good, but I'm not sure I specifically follow what you mean by 3. I think some elaboration on each of these points might be useful.
> 
> I think this checklist is great for running through to verify safety before flying. However, for all the points in this checklist, would it be beneficial to have an elaborated page of the safety guide? There, maybe pictures or elaboration could be useful for someone who has never used the system before. This could include the comment I made earlier about the meaning of "secured".

Referring back to my last comment on this section. I think this is more clear than the previous review, but I think I'm still a bit confused on all the details of 3. Maybe it's just because I haven't done this before, but I want to make sure others agree this looks good to go or if we need a bit of elaboration. Either in the form of a detailed checklist (maybe something that could even be printed to check off) or just some elaboration in the a-d points. What do you think @lowjunen @cacadam89 ?

**Review by exoticDFT — CHANGES_REQUESTED** [review]
Everything looks much better. But before we merge, let's get at least one of the other reviewers to double check the additions and comments.

File: docs/source/software/quick_start_guide.rst
Comment (k2shah) [reply docs/source/software/quick_start_guide.rst]: basically i just want the person to arm the quad on the ground, this will spin up the props but not enough for takeoff. this arming check is a good way to just see if the px4 system is in a good state (i.e. no sensor issues).

Comment (lowjunen) [reply docs/source/software/quick_start_guide.rst]: I think a check on the kill switch should be added. Something like:

PX4 Arms and Kills
a. Arm the quad by pinching the sticks down and inwards. *DO NOT* push up throttle stick after arm as this will cause the craft to take off.
b. gently push the throttle stick up until the motors start spinning. check that the propeller rotation is in the correct direction.
c. disarm the quad by pinching the sticks down and inwards.
d. Arm the quad again but this time, use the kill switch to disarm. The motors should cut instantaneously.

Note: For extra safety, you are advised to run these checks without the propellers attached.

**Comment by lowjunen** [comment]
yup! looks good to me

**Review by lowjunen — APPROVED** [review]
yup looks good to me!

**Review by exoticDFT — APPROVED** [review]
Looks good!

**Pull request merged**
